docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Convert an add-in to use the unified manifest for Microsoft 365
+description: Learn the various methods for converting an add-in with an XML manifest to the unified manifest for Microsoft 365 and sideload the add-in.
+ms.topic: how-to
+ms.date: 06/19/2023
+ms.localizationpriority: medium
+---
+
+# Convert an add-in to use the unified manifest for Microsoft 365
```

Comment:
I think our typical process in this scenario — when an entire article describes a feature in preview — is to add the preview tag to the h1 (example: https://learn.microsoft.com/en-us/office/dev/add-ins/outlook/one-outlook)
```suggestion
# Convert an add-in to use the unified manifest for Microsoft 365 (preview)
```